Description: match host arch as default
Author: Soren Hansen <soren@ubuntu.com>
Forwarded: no
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/344913

Index: libvirt-0.8.7/src/conf/capabilities.c
===================================================================
--- libvirt-0.8.7.orig/src/conf/capabilities.c	2010-12-20 07:35:22.000000000 -0600
+++ libvirt-0.8.7/src/conf/capabilities.c	2011-02-08 12:35:30.005095478 -0600
@@ -502,6 +502,13 @@
                                    const char *ostype)
 {
     int i;
+    /* Let's first see if we can match the host arch */
+    for (i = 0 ; i < caps->nguests ; i++) {
+        if (STREQ(caps->guests[i]->ostype, ostype) 
+              && STREQ(caps->guests[i]->arch.name, caps->host.arch))
+            return caps->guests[i]->arch.name;
+    }
+    /* If we couldn't we just grab the first match */
     for (i = 0 ; i < caps->nguests ; i++) {
         if (STREQ(caps->guests[i]->ostype, ostype))
             return 1;
